fonts/css/rswebfonts-ie7-codes.css

```diff
@@ -222,7 +226,7 @@
 .icon-srv-content-management { *zoom: expression( this.runtimeStyle['zoom'] = '1', this.innerHTML = '&#xe9ab;&nbsp;'); }
 .icon-srv-data-storage { *zoom: expression( this.runtimeStyle['zoom'] = '1', this.innerHTML = '&#xe9ad;&nbsp;'); }
 .icon-srv-database { *zoom: expression( this.runtimeStyle['zoom'] = '1', this.innerHTML = '&#xe9ae;&nbsp;'); }
-.icon-srv-design-build { *zoom: expression( this.runtimeStyle['zoom'] = '1', this.innerHTML = '&#xe9af;&nbsp;'); }
+.icon-srv-design-build-2 { *zoom: expression( this.runtimeStyle['zoom'] = '1', this.innerHTML = '&#xe9af;&nbsp;'); }
```

Comment:
@amberRrucker 
This shouldn't matter as its only part of the css generated by fontello and never gets consumed by zoolander or www. The important part is here https://github.com/brantdarilek/zoolander/blob/rsweb-9084-icon-font/styleguide/_themes/global/scss/rsweb-font-style.scss#L359 which is were we specify the icon in our styles which actually do get consumed.